Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityToolkitConfigurationProfile.cs

```diff
@@ -36,6 +36,20 @@ public class MixedRealityToolkitConfigurationProfile : BaseMixedRealityProfile
 
         #region Mixed Reality Toolkit configurable properties
 
+        [EnumFlags]
+        [SerializeField]
+        [Tooltip("The types of debug messages to log to the output window.")]
+        private LoggingLevels loggingLevel = (LoggingLevels)(-1);
+
+        /// <summary>
+        /// The desired types of debug messages to log to the output window.
+        /// </summary>
+        public LoggingLevels LoggingLevel
```

Comment:
What if users want to supply their own custom logging levels?

Reply:
> What if users want to supply their own custom logging levels?

Are you asking about runtime users or developer users? Developers can set their desired level in the MixedRealityToolkitConfigurationProfile and that will control the core toolkit object and the services.

If you are asking about custom application code, this is why there are DebugUtilities overloads that take the logging level as a parameter.  It allows any code to send custom levels to the log methods.

Reply:
Sounds good. Let's also add that virtual property to the `IMixedRealityService` interface, and update each Service profile with the logging level. 

For the extension services we can add the logging level right under the Runtime Platform settings.

Reply:
> Let's also add that virtual property to the IMixedRealityService interface, and update each Service profile with the logging level.

Only concern here is that customers will be required to change the logging level for ALL of the services individually.... that can be quite a few.  Any thoughts on a good way to have a "set all to <level>" feature? Perhaps on the toolkit menu?

Reply:
> Let's also add that virtual property to the IMixedRealityService interface,

Propose making this change in a subsequent PR (when the services are updated to honor the setting). Sound fair?

Reply:
Oh, I see that was your part 2 PR?

Reply:
yep :)